net-misc/streamlink/streamlink-0.8.1.ebuild

```diff
@@ -24,6 +24,7 @@ RDEPEND="dev-python/pycryptodome[${PYTHON_USEDEP}]
 	dev-python/backports-shutil_which[$(python_gen_usedep 'python2*')]
 	dev-python/backports-shutil_get_terminal_size[$(python_gen_usedep 'python2*')]
 	dev-python/pycountry[${PYTHON_USEDEP}]
+	dev-python/websocket-client[${PYTHON_USEDEP}]
```

Comment:
please revbump this ebuild (due to the required change to the VDB)

Reply:
done